src/wkt/src/float.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// A helper to serialize `f32` to ProtoJSON format.
+// TODO(#1767): use float_serialize from generated code.
+#[allow(dead_code)]
+fn float_serialize<S>(x: &f32, s: S) -> std::result::Result<S::Ok, S::Error>
```

Comment:
I think:

```suggestion
pub fn float_serialize<S>(x: &f32, s: S) -> std::result::Result<S::Ok, S::Error>
```

should work.

Reply:
This + making the module public worked.